recipes/pyebsdindex/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "pyebsdindex" %}
+{% set version = "0.1rc2" %}
```

Comment:
I missed this one but a release candidate falls under: https://github.com/conda-forge/cfep/blob/main/cfep-05.md

Reply:
Did not know this, thanks for informing me of the prerelease specification for conda-forge.

PyEBSDIndex has only prereleases [available from PyPI](https://pypi.org/project/pyebsdindex/#history), and according to the [Exceptions part](https://pypi.org/project/pyebsdindex/#history) of the specification, packages without non-prereleases should *not* publish to a prerelease label. Correct?

Reply:
To be fair that exception was made for black b/c it was a popular package that never had an official release up until recent. Some people find it funny to make it a forever beta/alpha product, I guess it reminds then on 1995s Google.

Anyway, your options now are:

- make a case that your package is as important as black to the community and that it must be in main channel
- add a conda_build_config.yaml and release it under a dev channel
- ask upstream for a stable release

Reply:
> that exception was made for black b/c it was a popular package that never had an official release up until recent

Thanks for the clarification. I understand that such an exception was made for black. I did *not* read the "Exceptions" part as to pertain only to *popular* packages, but every package with nothing else than prererelases. I might suggest the a more clear wording than "Certain packages" and "such a package".

I've adopted your second option.

Reply:
Indeed the text leaves it open for more exceptions. That is the problem with exceptions, right!

Remember to remove that file and rerender when we have a stable release.